content/docs/ui/managing-contacts/segmenting-your-contacts.md

```diff
@@ -164,17 +164,28 @@ The following tables list the available fields, excluding custom fields that you
   </tr>
 </table>
 
-\* Single Send Activity fields have their own Single Send type. The operators available for a Single Send ("was sent," "opened," and "clicked") behave like [Email Activity fields](#email-activity-fields), but they are scoped to engagement on Single Sends.
+### Automations fields
+
+<table>
+  <tr>
+    <th>Field Name</a></th>
+    <th>Field Type</a></th>
+  </tr>
+  <tr>
+    <td>Automation Activity</td>
+    <td>Automation*</td>
+  </tr>
+</table>
+
+\* Single Send Activity and Automation Activity fields have their own Single Send and Automation types. The operators available for a Single Send or Automation ("was sent," "has clicked", and "has opened") behave like [Email Activity fields](#email-activity-fields), but they are scoped to engagement on Single Sends or Automations.
```

Comment:
Single Send Activity and Automation Activity fields have their own Single Send and Automation types. You’ll then choose whether the operator applies to any Single Send/Automation or a specific Single Send/Automation. The last step will be to define the time period the engagement occurred within.